generics/constants/endpoints.js

```diff
@@ -57,4 +57,6 @@ module.exports = {
 	FIND_ENTITY_DOCUMENTS: '/v1/entities/find',
 	FIND_ENTITY_TYPE_DOCUMENTS: '/v1/entityTypes/find',
 	ORGANIZATION_READ: '/v1/organization/read',
+	USER_SERVICE_URL: 'http://localhost:3001/user/v1/user/read/',
+	ENTITY_MANAGEMENT_SERVICE_URL: 'http://localhost:5001/entity/v1/entities/find',
```

Comment:
@MallanagoudaB base url we should take from env. here you only have to mention endpoint. And it is already defined here check : FIND_ENTITY_TYPE_DOCUMENTS